package.json

```diff
@@ -22,5 +22,10 @@
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/github": "^5.1.1"
+  },
+  "devDependencies": {
+    "@zeit/ncc": "^0.22.3",
```

Comment:
Why did you change @vercel/ncc to @zeit/ncc? The package from @zeit has been deprecated in favor of @vercel

```
npm WARN deprecated @zeit/ncc@0.22.3: @zeit/ncc is no longer maintained. Please use @vercel/ncc instead.
```

Reply:
Because `@zeit/ncc` [worked](https://github.com/deemp/cache-nix-too/blob/eefa7c9691f7e13478dab17d58402e144d5bb07e/package.json#L37C7-L37C7) for me in another GH Action... and because I didn't notice you used `@vercel/ncc`. Switched to the latest `@vercel/ncc`